src/packet_analysis/protocol/ethernet/Ethernet.cc

```diff
@@ -62,25 +53,21 @@ bool EthernetAnalyzer::AnalyzePacket(size_t len, const uint8_t* data, Packet* pa
 			return false;
 			}
 
-		// Let specialized analyzers take over for non Ethernet II frames.
-		// Note that pdata remains at the start of the ethernet frame.
+		len -= 14;
+		data += 14;
```

Comment:
This is skipping the source address, destination address, and the length field to get to the start of the LLC header. That's 14 bytes, unless I'm missing something?

Reply:
Nevermind, I got confused by an [overview](https://www.ionos.de/digitalguide/server/knowhow/ethernet-frame/) that made it look like in case of SNAP the last two bytes describe a type instead of the length.